src/server/services/buurt/datasets.ts

```diff
@@ -423,7 +468,13 @@ function transformMeldingenBuurtResponse(
 
       const displayCat = capitalizeFirstLetter(categorie);
 
-      if (config.idKeyList) feature.properties[config.idKeyList] = index;
+      if (config.idKeyList) {
+        feature.properties[config.idKeyList] = slug(
+          feature.properties.category.slug +
+            '-' +
+            feature.properties.created_at.replaceAll('-', '')
+        );
+      }
```

Comment:
slugme zie ik vaak in de codebase. Maar dan zonder dat het gebruikt voor. out of scope maar deze plugin is misschien interessant? eslint-plugin-unused-imports